circleci: failed tests: TestClusterRecovery

The following test appears to have failed:

[#18804](https://circleci.com/gh/cockroachdb/cockroach/18804):

```
W160608 18:06:51.807605 acceptance/chaos_test.go:247  round 1: cluster recovered
I160608 18:06:51.907979 acceptance/cluster/localcluster.go:658  stopping
I160608 18:06:51.908218 acceptance/cluster/localcluster.go:556  event stream done, resetting...: net/http: request canceled
I160608 18:06:51.908236 acceptance/cluster/localcluster.go:579  events monitor exits
I160608 18:06:52.623788 acceptance/chaos_test.go:174  client 1 shutting down
--- FAIL: TestClusterRecovery (10.09s)
	chaos_test.go:265: pq: context deadline exceeded
=== RUN   TestNodeRestart
I160608 18:06:53.655107 acceptance/cluster/localcluster.go:289  Initializing Cluster AdHoc 3x1:
{"name":"AdHoc 3x1","nodes":[{"count":3,"stores":[{"count":1,"max_ranges":0}]}],"duration":5000000000}
I160608 18:06:53.656819 acceptance/cluster/docker.go:105  ImagePull cockroachdb/builder:20160526-162455 already exists
I160608 18:06:54.258264 acceptance/cluster/localcluster.go:590  creating certs (1024bit) in: /home/ubuntu/cockroach/.localcluster.certs.272371237
I160608 18:06:54.311079 acceptance/cluster/localcluster.go:541  events monitor starts
I160608 18:06:54.549826 acceptance/cluster/localcluster.go:490  *** started roach0 ***
  ui:        https://127.0.0.1:32795
  trace:     https://127.0.0.1:32795/debug/requests
--
E160608 18:09:36.786466 acceptance/cluster/localcluster.go:506  node=2 status=restart
I160608 18:09:48.769463 acceptance/cluster/localcluster.go:658  stopping
I160608 18:09:48.769706 acceptance/cluster/localcluster.go:556  event stream done, resetting...: net/http: request canceled
I160608 18:09:48.769723 acceptance/cluster/localcluster.go:579  events monitor exits
--- PASS: TestRaftUpdate (22.47s)
FAIL
ok github.com/cockroachdb/cockroach/acceptance 1337s

```
Please assign, take a look and update the issue accordingly.

--- a/acceptance/dynamic_client.go
+++ b/acceptance/dynamic_client.go
@@ -68,7 +68,7 @@ func (dc *dynamicClient) Close() {
 // or an issue with the node itself. This can occur when a node is restarting
 // or is unstable in some other way.
 func isRetryableError(err error) bool {
-	return testutils.IsError(err, "(connection reset by peer|connection refused|failed to send RPC|EOF)")
+	return testutils.IsError(err, "(connection reset by peer|connection refused|failed to send RPC|EOF|context deadline exceeded)")
 }
 
 var errTestFinished = errors.New("test is shutting down")

